### ISSUE TITLE
alt key does not work at start up

### ISSUE THREAD

**Issue by Tubbles** [microsoft/calculator#328]
**Describe the bug**
When I start calc the alt-key and all keyboard shortcut combinations based off of it wont work.

**Steps To Reproduce**
1. Start calc, I do via Win+R and type calc
2a. Press alt (view accessible shortcuts)
2b. Press alt+1 (switch to standard mode)
2c. Press alt+2 (switch to scientific mode)
2d. Press alt+3 (switch to programmer mode)

**Expected behavior**
Functionality according to the specified hotkey in 2a-d above.

**Actual behavior**
The steps 2a-d above does not do anything.

**Device and Application Information (please complete the following information):**
 - OS Build: Windows 10 Enterprise 15063.1689 (version 1703)
 - Architecture: x64
 - Application Version: 10.1702.312.0

**Note:**
The described fault mode disappears if I select one of the modes using the mouse instead. After doing it with the mouse at least once since calc starts, the shortcuts described above starts to work. Opening and closing the hamburger menu also seems to do the trick.

**Comment by grochocki** [comment]
Thanks for the report! I am not able to reproduce this issue, though. I can switch between modes with alt+1/2/3/4 access keys. I am on a newer OS/app version combo:

OS Build: 177763.379 (version 1809)
Application Version: 10.1902.20.0

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by rudyhuyn** [comment]
> Thanks for the report! I am not able to reproduce this issue, though. I can switch between modes with alt+1/2/3/4 access keys. I am on a newer OS/app version combo:
> 
> OS Build: 177763.379 (version 1809)
> Application Version: 10.1902.20.0

Same, I was not able to reproduce this issue. Probably something related to Windows 10 1703 or the old version of the app.

![DA24AE01-0C23-47C7-8867-878592389081](https://user-images.githubusercontent.com/1226538/54548860-0f146f00-4966-11e9-86c6-bf985905ceed.GIF)

**Comment by grochocki** [comment]
We are still not able to reproduce this issue, so closing for now, but will revisit if we discover a way to reproduce this more reliably.
